### PR TITLE
Add nvdimm command

### PR DESCRIPTION
**Pull request by rvykydal**
Related: rhbz#1499442

### CODE REVIEW

File: tests/commands/nvdimm.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                          "nvdimm reconfigure --namespace=namespace0.0 --mode=sector --sectorsize=512\n")
+        self.assert_parse("nvdimm reconfigure --namespace=whatever --mode=sector --sectorsize=512",
+                          "nvdimm reconfigure --namespace=whatever --mode=sector --sectorsize=512\n")
+
+        # fail
+        # --sectorsize is required when recofiguring to sector --mode
+        self.assert_parse_error("nvdimm reconfigure --namespace=namespace0.0 --mode=sector")
+        self.assert_parse_error("nvdimm reconfigure --namespace=namespace0.0 --mode=invalid --sectorsize=512")
+        self.assert_parse_error("nvdimm reconfigure --mode=sector --sectorsize=512")
+        #self.assert_parse_error("nvdimm reconfigure --namespace=namespace0.0 --mode=sector --sectorsize=512")
```
Comment (poncovka) [inline tests/commands/nvdimm.py]: If this line is useful to keep, please add a comment with an explanation.

File: pykickstart/commands/nvdimm.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        if not y:
+            return False
+        return (self.action == y.action and
+                self.namespace == y.namespace)
+
+    def __ne__(self, y):
+        return not self == y
+
+    def _getArgsAsStr(self):
+
```
Comment (jkonecny12) [inline pykickstart/commands/nvdimm.py]: You don't have this blank line anywhere else. Could you please remove it here too?

File: pykickstart/commands/nvdimm.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            if self.mode == NVDIMM_MODE_SECTOR:
+                retval += " --mode=%s" % NVDIMM_MODE_SECTOR
+                if self.sectorsize:
+                    retval += " --sectorsize=%d" % self.sectorsize
+
+        return retval
+
+    def __str__(self):
+        retval = BaseData.__str__(self)
+        retval += ("nvdimm %s" % self._getArgsAsStr())
```
Comment (jkonecny12) [inline pykickstart/commands/nvdimm.py]: I don't think you need those brackets here `()`.

**Comment by rvykydal** [comment]
I've added one more patch for "nvdimm use" command

File: pykickstart/commands/nvdimm.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+        op = KSOptionParser(prog="nvdimm", version=F28, description="""
+                            Perform an action on an nvdimm device.""")
+        op.add_argument("action", choices=self.validActions, nargs=1, default=NVDIMM_ACTION_RECONFIGURE,
+                        version=F28, help="""
+                        The action to be performed on the device specified by further options.
+                        The device is specified by ``--namespace`` option.
+
+                        Valid actions:
+                        - ``reconfigure``: Reconfigures the device into the mode specified by
+                                           ``--mode`` and (depending on the mode) ``--sectorisize`` options.
```
Comment (clumens) [inline pykickstart/commands/nvdimm.py]: Typo here - `--sectorisize` should be `--sectorsize`

**Comment by clumens** [comment]
Bah, there's no need to go around again to fix one typo.  I'll merge this and then fix that myself.

**Pull request merged**